index.bs

```diff
@@ -2221,21 +2241,28 @@ When this method is invoked, the user agent MUST execute the following algorithm
                                     configuration knowledge of the appropriate transport to use with |authenticator| in making its
                                     selection.
 
+                                    <!-- @@EDITOR-ANCHOR-01: KEEP THIS LIST SYNC'D WITH OTHER LOCATIONS WITH THIS TAG -->
                                     Then, using |transport|, invoke the [=authenticatorGetAssertion=] operation on
                                     |authenticator|, with |rpId|, |clientDataHash|, |allowCredentialDescriptorList|,
-                                    |userVerification|, and |authenticatorExtensions| as parameters.
+                                    |userVerification|,
+                                    |attestationFormats|,
+                                    and |authenticatorExtensions| as parameters.
 
+                                    <!-- @@EDITOR-ANCHOR-01: KEEP THIS LIST SYNC'D WITH OTHER LOCATIONS WITH THIS TAG -->
                                 :   [=list/is empty=]
                                 ::  Using local configuration knowledge of the appropriate transport to use with |authenticator|,
                                     invoke the [=authenticatorGetAssertion=] operation on |authenticator| with |rpId|,
-                                    |clientDataHash|, |allowCredentialDescriptorList|, |userVerification|, and
-                                    |authenticatorExtensions| as parameters.
+                                    |clientDataHash|, |allowCredentialDescriptorList|, |userVerification|,
+                                    |attestationFormats|, and |authenticatorExtensions| as parameters.
                             </dl>
 
+                    <!-- @@EDITOR-ANCHOR-01: KEEP THIS LIST SYNC'D WITH OTHER LOCATIONS WITH THIS TAG -->
                     :   [=list/is empty=]
                     ::  Using local configuration knowledge of the appropriate transport to use with |authenticator|, invoke the
                         [=authenticatorGetAssertion=] operation on |authenticator| with |rpId|, |clientDataHash|,
-                        |userVerification| and |authenticatorExtensions| as parameters.
+                        |userVerification|,
+                        |attestationFormats|,
+                        and |authenticatorExtensions| as parameters.
```

Comment:
These changes currently conflict with `main` because of the new [§5.1.4.2. Issuing a Credential Request to an Authenticator](https://w3c.github.io/webauthn/#sctn-issuing-cred-request-to-authenticator) section. In the merge these changes need to be moved to §5.1.4.2, and the `|attestationFormats|` parameter needs to be added as well.